platforms/hyperledger-fabric/configuration/roles/create/ca-tools/orderer/tasks/main.yaml

```diff
@@ -146,10 +202,13 @@
 ############################################################################################
 # This task copies the tls ca.crt file from the respective CA Tools CLI
 # to the address specified in network.yaml
-- name: Copy the tls ca.crt file from the ca tools
+- name: Copy the tls ca.crt file from vault
```

Comment:
Please rename the task and the above comment to `Fetch the tls ca.crt from vault`